lib/kernel/async_work_group_strided_copy.cl

```diff
@@ -0,0 +1,77 @@
+/* OpenCL built-in library: async_work_group_copy()
+
+   Copyright (c) 2012 Pekka Jääskeläinen / Tampere University of Technology
```

Comment:
If you have used my original code as a basis and updated it for the striding, please add your name here too and updated year. Also the file name above is wrong.